docs/index-utils.md

```diff
@@ -36,6 +36,13 @@ Converts a collection lookupDocid (in the raw corpus) to a Lucene internal looku
 Converts to a Lucene internal lookupDocid to a collection lookupDocid  (default: 0)
 
 
+```
+-dumpAllDocids outputCompressionFormat
+```
+Dumps all Docids in the index. For non-tweets collection the order is in ascending of String docid;
```

Comment:
you have Docid here and docid below - I think we've used mostly docid (lowercase)?

Reply:
okay